python/cudf/tests/test_sorting.py

```diff
@@ -166,3 +167,50 @@ def test_dataframe_nsmallest_sliced(counts, sliceobj):
     gdf = DataFrame.from_pandas(df)
     got = gdf[sliceobj].nsmallest(n, 'a')
     assert (got.to_pandas() == expect).all().all()
+
+
+@pytest.mark.parametrize('num_cols', [1, 2, 3, 5])
+@pytest.mark.parametrize('num_rows', [0, 1, 2, 1000])
+@pytest.mark.parametrize(
+    'dtype',
+    ['int8', 'int16', 'int32', 'int64', 'float32', 'float64',
+     'datetime64[ms]']
+)
+@pytest.mark.parametrize('nulls', ['none', 'some', 'all'])
+@pytest.mark.parametrize('ascending', [True, False])
+@pytest.mark.parametrize('na_position', ['first', 'last'])
+def test_dataframe_multi_column(num_cols, num_rows, dtype, nulls, ascending,
+                                na_position):
+    if dtype not in ['float32', 'float64'] and nulls in ['some', 'all']:
+        pytest.skip(msg='nulls not supported in dtype: ' + dtype)
```

Comment:
If we're going to lean on numpy / pandas to compare, nulls in non-float dtypes aren't supported which is why I skipped these tests. I could split the float tests to a separate function to avoid all the skips if we wanted, but I'm not sure of an easy way to handle the sorting + equality checking  otherwise.